src/server/plugins/pageinfo.ts

```diff
@@ -0,0 +1,56 @@
+import { ParsedContent } from '@nuxt/content/dist/runtime/types'
+import fs from 'fs'
+import os from 'os'
```

Comment:
好みの問題かもですけど…

> npmからインストールしたサードパーティ製のモジュールとの区別が明確になるため、付けておくことが推奨されます。

https://jsprimer.net/use-case/nodecli/read-file/#fs-module

```suggestion
import fs from 'node:fs'
import os from 'node:os'
```